Client/src/components/common/SessionTimeoutHandler.jsx

```
@@ -0,0 +1,118 @@
+                    onClick={logout}
+                    className="inline-flex items-center px-3 py-1.5 border border-gray-300 text-xs font-medium rounded-md text-gray-700 bg-white hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-blue-500"
+                  >
+                    Logout now
+                  </button>
+                </div>
+              </div>
+              <div className="ml-4 flex-shrink-0 flex">
+                <button
+                  className="inline-flex text-gray-400 hover:text-gray-500"
+                  onClick={() => {}}
+                >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)